parts/windows/kuberneteswindowssetup.ps1

```diff
@@ -353,6 +362,12 @@ try
         New-CsiProxyService -CsiProxyPackageUrl $global:CsiProxyUrl -KubeDir $global:KubeDir
     }
 
+    if ($global:EnableSecureTLSBootstrapping) {

```

Comment:
It seems like that it only downloads the binary in cse in node provisioning. So it may still fail in running `tls-bootstrap-client.exe` after cse returns success. Is it by design?

Reply:
also referenced here: https://learn.microsoft.com/en-us/azure/aks/kubelogin-authentication